internal/controller/entra/main.go

```diff
@@ -134,41 +143,59 @@ func (c *Controller) IsStaffMember(ctx context.Context, username types.Username)
 
 func (c *Controller) SendInvite(ctx context.Context, email string, sponsor types.Sponsor) error {
 
-	// check if user exists in entra, if yes, don't send invite
+	// check if user exists in entra, if not, send an invite; if yes, custom notification
 	user, err := c.userData(ctx, types.Username(email))
-	if err != nil {
+	if err != nil && strings.Contains(err.Error(), "does not exist") {
```

Comment:
this error is wrapped already so we can do
```suggestion
	if err != nil && errors.Is(err, types.ErrNotFound) {
```
and only have to do the string match in one place